libs/libprom/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2010-2014 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libprom
+PKG_VERSION:=0.1.3
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+PKG_SOURCE_URL:=https://github.com/digitalocean/prometheus-client-c.git
```

Comment:
How this is relevant to libprom? 🤔

Reply:
The project consists of two C libraries - `libprom` and `libpromhttp` in `prom` and `promhttp` subdirs.

And original packaging names them as libprom: https://github.com/digitalocean/prometheus-client-c/blob/master/prom/CMakeLists.txt#L129